src/api-endpoints.ts

```diff
@@ -277,6 +278,37 @@ export const databasesCreate = {
   path: () => `databases`,
 } as const
 
+/*
+ * databases.update()
+ */
+
+interface DatabasesUpdatePathParameters {
+  database_id: string
+}
+interface DatabasesUpdateQueryParameters {}
+
+export type UpdatePropertySchemaMap = {
+  [propertyName: string]: UpdatePropertySchema
+}
+interface DatabasesUpdateBodyParameters {
+  properties?: UpdatePropertySchemaMap
+  title?: RichTextInput[]
+}
+
+export interface DatabasesUpdateParameters
+  extends DatabasesUpdatePathParameters,
+    DatabasesUpdateQueryParameters,
+    DatabasesUpdateBodyParameters {}
+export interface DatabasesUpdateResponse extends BlockBase {}
```

Comment:
This returns a `Database` object right? not `BlockBase`?

Reply:
good catch, fixed!